tutor4points/core/templates/users_profile.html

```diff
@@ -175,4 +226,12 @@
     }
 </script>
 
+{{ success_message | json_script:"success-message" }}
```

Comment:
The success message seems to appear all the time on the tutors page once one form is successfully submitted.